src/snowflake/cli/api/commands/snow_typer.py

```diff
@@ -105,7 +105,7 @@ def command_callable_decorator(*args, **kw):
                     result = command_callable(*args, **kw)
                     self.process_result(result)
                     execution.complete(ExecutionStatus.SUCCESS)
-                except Exception as err:
+                except BaseException as err:
```

Comment:
Wondering if it might be simpler to move the `execution.complete()` calls to the finally block instead? That way you don't need to worry about catching the wrong exceptions, and you guarantee calling the logic to set the exit time

Reply:
There are other stuff that need to be handled during the exception, like setting complete with a failure status, and running the exception handler. This way, we ensure it goes in the right path during the exception (not only fix the time)